app/.storybook/preview.js

```diff
@@ -23,6 +23,22 @@ const parameters = {
   },
   // Configure i18next and locale/dropdown options.
   i18n,
+  options: {
+    storySort: {
+      method: "alphabetical",
+      order: [
+        "Welcome",
+        "Core",
+        // Storybook infers the title when not explicitly set, but is case-sensitive
+        // so we need to explicitly set both casings here for this to properly sort.
+        "Components",
+        "components",
+        "Templates",
+        "Pages",
+        "pages",
+      ],
```

Comment:
question: so is the idea that the top level is a manual order, then levels nested under that are sorted alphabetically?

Reply:
Yea exactly

> Note that the order option is independent of the method option; stories are sorted first by the order array and then by either the method: 'alphabetical' or the default configure() import order.

https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#sorting-stories